docs/content/docs/plugins/extending-the-admin-ui/_index.md

```diff
@@ -89,3 +89,8 @@ plugins: [
   }),
 ],
 ```
+
+## Avaliable custom settings
+
+* [Adding UI Translations]({{< relref "adding-ui-translations" >}})
```

Comment:
Hi, i found out that theres is a translation guide folder without link anywhere in the docs, so i added here, but i don'k know if i should have done.

Reply:
We don't actually need a manual link to that because Hugo generates a menu item in the docs nav menu, and also a "contents" link in the parent page. See https://www.vendure.io/docs/plugins/extending-the-admin-ui/

Reply:
So i will remove the changes on this file